tracing-fmt/src/writer.rs

```diff
@@ -0,0 +1,165 @@
+//! Abstractions for creating [`io::Write`] instances.
+
+use super::Builder;
+use std::io;
+
+/// A type that can create [`io::Write`] instances.
```

Comment:
These links to types from the standard library are not being rendered:
<img width="879" alt="Screen Shot 2019-08-25 at 10 34 08 AM" src="https://user-images.githubusercontent.com/2796466/63653620-15ccbc80-c724-11e9-80e6-8edc3f3f3757.png">

Can we change this to putting the link URLs at the bottom of the doc comments? I believe that will fix this.

Reply:
Ah my apologies, I was testing the documentation against nightly and forgot that stable doesn't have the convenience linking feature yet.

Reply:
Yeah, unfortunately the published RustDoc will be built on stable by docs.rs.

Reply:
I'll still fix this, but [according to docs.rs], they actually use nightly:

> Docs.rs automatically builds crates' documentation released on crates.io using the nightly release of the Rust compiler. The current version of the Rust compiler in use is rustc 1.39.0-nightly (bea0372a1 2019-08-20).

It's still a good idea to fix it for anyone using `cargo doc` locally though, since (as you mentioned) intra-rustdoc links are still being fleshed out.

[according to docs.rs]: https://docs.rs/about

Reply:
> I'll still fix this, but [according to docs.rs](https://docs.rs/about), they actually use nightly:

Oh, huh, I hadn't realized that. I think local `cargo doc` gets enough use that it's still worth making sure it renders correctly, though.